unwind/src/glue.rs

```diff
@@ -1,7 +1,13 @@
 use gimli::X86_64;
-use super::{UnwindPayload, StackFrames};
 use registers::Registers;
 
+type UnwindPayload<'a> = &'a mut dyn FnMut(Registers);
+
+pub fn registers<F>(mut f: F) where F: FnMut(Registers) {
+    let mut f = &mut f as UnwindPayload;
```

Comment:
Edge case: the closure itself might be `Drop`, right? Is this a problem (At a glance I don't see one)?

Reply:
I think for the purposes of this module, `Drop` is fine. If the caller needs to restrict that then it's up to them.